### PR TITLE
Create post

### PR DESCRIPTION
**Pull request by tanishk15sharma**
- [x] Style: create post component
- [x] `useState` for inputs
- [x] upload post image ion cloud
- [x] `POST` cloud API
- [x] `POST` new post with image `url`
- [x] `Refactor` API calls to utils

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
clean and modular code 
API handled perfectly 
looks good to merge

**Comment by tanishk15sharma** [comment]
> clean and modular code API handled perfectly looks good to merge

File: src/utils/posts.js [hunk 1]
```diff
@@ -50,4 +50,51 @@ const likeDislikePost = async (postId) => {
...
+    uploadData.append("cloud_name", "tanishkcloudimg");
+    const { data, status } = await axios.post(
+      "https://api.cloudinary.com/v1_1/tanishkcloudimg/image/upload",
+      uploadData
+    );
+    if (status === 200) {
+      return data.secure_url;
+    }
+  } catch (err) {
+    console.log(err);
```
Comment (znjs) [inline src/utils/posts.js:67]: can use `console.error`

File: src/utils/posts.js [hunk 2]
```diff
@@ -50,4 +50,51 @@ const likeDislikePost = async (postId) => {
...
+        headers: {
+          token,
+        },
+      }
+    );
+    if (status === 200) {
+      return data;
+    }
+  } catch (err) {
+    console.log(err);
```
Comment (znjs) [inline src/utils/posts.js:90]: can use `console.error`

**Review by znjs — APPROVED** [review]
- Clean, modular code
- easy to understand
- Good PR

File: src/utils/posts.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/utils/posts.js:67]: yes , actually I will use toasts later on .

File: src/utils/posts.js (hunk 2 above)
Comment (tanishk15sharma) [reply src/utils/posts.js:90]: thank you for your comment.

**Pull request merged**